linkerd/app/src/spire.rs

```diff
@@ -11,8 +11,8 @@ const TONIC_DEFAULT_URI: &str = "http://[::]:50051";
 
 #[derive(Clone, Debug)]
 pub struct Config {
-    pub(crate) socket_addr: Arc<String>,
-    pub(crate) backoff: ExponentialBackoff,
+    pub socket_addr: Arc<String>,
+    pub backoff: ExponentialBackoff,
```

Comment:
This is an unrelated change. The main reason is to unify visibility of types used in the identity config (e.g control::Config fields are marked as pub).